lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -86,6 +86,15 @@
     choices: ["yes", "no"]
     aliases: []
 
+  installroot:
+    description:
+      - Specifies an alternative installroot, relative to which all packages
+        will be installed.
+    required: false
+    version_added: "2.3"
+    default: "/"
+    aliases: []
```

Comment:
As this is empty you can remove this.

Could you please remove them from the other docs above.